src/ui/theme.tsx

```diff
@@ -201,6 +192,30 @@ export function createGetViewPortConfig(params: {
     return { getViewPortConfig };
 }
 
+function getPalette(themeId: typeof THEME_ID, paletteOverrides?: Object): PaletteBase {
```

Comment:
Please use a cosure, we prefer avoiding creating function that are used only once.